lib/ClangImporter/ClangImporter.cpp

```diff
@@ -68,7 +68,9 @@ using namespace importer;
 using clang::CompilerInstance;
 using clang::CompilerInvocation;
 
+#if !defined(_MSC_VER)
 #pragma mark Internal data structures
+#endif
```

Comment:
This is pretty awful. Does MSVC really not ignore pragmas it doesn't understand? If so, maybe we should just use comments instead. @DougGregor, @milseman, any opinions?